### DISCUSSION TITLE
How to separate two Stream Deck XL Surfaces that are mirrored

### DISCUSSION THREAD

**Discussion by keith32e** [bitfocus/companion#2872] [Q&A]
I am new to Companions, so please forgive if already addressed. 

I have two Stream Deck XL Surfaces that I  want to set up. One has been used with the Stream Deck app to set up PTZ camera position presets, and has been working well with OBS. I am preparing to move back to vMix, and added a second XL for PreView/Program switching, transitions,  and multi layer/box shots. 

However, Companion sees both Surfaces,  but mirrors them to the same button layouts. I tried removing the Stream Deck app, removing Companions, rebooting, and reinstalling Companions with the XLs unplugged, but they were still in the configuration when I reinstalled Companion, even though unplugged. In fact, the vMix IP Address was also remembered. 

Is there a file or directory or registry entry that is still there? Is there a command to split the XLs into separate logical Surfaces? I see identifiers for each, and can give them different Names, but they are still acting as a mirror set. There must be something that I am missing. 
Thanks for your great work.

**Comment by dnmeid** [comment]
This is a two part question
1. There is no real coexistance possible between the Stream Deck app and Companion, because the Elgato app always uses all connected Stream Decks when run. You can either use only Companion or only the Elgato app. You can use the Companion plugin within the Elgato app if you want to have both, but it has some limitations.
2. If you connect two or more Stream Decks / surfaces to Companion they will all share the same set of pages and by default will all be positioned with the top left corner at 0/0. You can simply switch one of the surfaces to a different page and then they do different things, all surfaces can show an idividual page. You can also make the button grid larger and give one of your surfaces a different position on the grid. You can also put several surfaces in a group, so you can easily keep them on the same page.
